lib/Serialization/ModuleFileCoreTableInfo.h

```diff
@@ -330,7 +330,7 @@ class ModuleFileSharedCore::DeclMembersTableInfo {
   }
 
   hash_value_type ComputeHash(internal_key_type key) {
-    return llvm::hash_value(key);
+    return key;
```

Comment:
`hash_value` ends up calling `getHashValue` and for a `uint32_t` that was just `val * 37`. Happy to change it to that, to reinterpret_cast then `djbHash`, or just leave it like this - let me know!